docs/sphinx/BUILD.bazel

```diff
@@ -83,7 +85,13 @@ sphinx_stardocs(
         "api/entry_points/py_console_script_binary.md": "//python/entry_points:py_console_script_binary_bzl",
         "api/packaging.md": "//python:packaging_bzl",
         "api/pip.md": "//python:pip_bzl",
-    },
+    } | ({
+        # Bazel 6 + Stardoc isn't able to parse something about the python bzlmod extension
+        "api/extensions/python.md": "//python/extensions:python_bzl",
+    } if IS_BAZEL_7_OR_HIGHER else {}) | ({
+        # This depends on @pythons_hub, which is only created under bzlmod
+        "api/extensions/pip.md": "//python/extensions:pip_bzl",
```

Comment:
nit: I don't see the docs in the PR preview here: https://rules-python--1629.org.readthedocs.build/en/1629/api/extensions/python.html#

Reply:
Good catch! The RTD builds weren't using --config=rtd. Fixed.